storm-core/src/native/worker-launcher/impl/utils/cJSON.c

```diff
@@ -0,0 +1,2932 @@
+/*
+  Copyright (c) 2009-2017 Dave Gamble and cJSON contributors
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
do we need to adjust any licensing due to this?

Reply:
I don't think so. Hadoop uses this the same way